### DISCUSSION TITLE
AssertionError: PyAPS: Mask array size mismatch

### DISCUSSION THREAD

**Discussion by laxmangeo** [insarlab/MintPy#1267] [Q&A]
Hi sir,
          I got an error at the step - correct troposphere.
![mintpy](https://github.com/user-attachments/assets/6c2878a1-3249-40aa-a21b-a831ca3f90eb)
how can I solve this?

**Comment by ditafaith** [comment]
Hi @laxmangeo, just to make sure, have you setup the .cdsapirc in your home directory?

**Comment by laxmangeo** [comment]
yes.

**Comment by yunjunz** [comment]
@laxmangeo you may want to modify your response and do not show this private account info of yours.
